weather.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+require("dotenv").config();
+const WEATHER_API_KEY = process.env.WEATHER_API_KEY;
+const axios = require("axios");
+
+// this should use find()
+let haveWeatherData = (city) => {
```

Comment:
Nice work on the refactor so far! 

Feedback here: Consider using `const` vs let for your `haveWeatherData` and `getWeather` functions as the usage becomes more clear, and if you don't want to reassign these, then we might as well use const.